packages/mobile/src/notifications.ts

```diff
@@ -79,6 +83,16 @@ class PushNotifications {
     }
   }
 
+  // Method used to open the push notification that the user pressed while the app was closed
+  popInitial = () => {
```

Comment:
so this one pops and also opens, so maybe something like `openInitialNotification` or onInitialNotification or smthg?